docs/dev/tools/cli.md

```diff
@@ -669,14 +669,14 @@ This includes:
 
 ```sh {class="command-line" data-prompt="$"}
 viam machines list
-viam machines status --organization=<org id> --location=<location id> --machine=<machine id>
-viam machines logs --organization=<org id> --location=<location id> --machine=<machine id> [...named args]
-viam machines api-key create --machine=<machine id> [...named args]
-viam machines part list --machine=<machine-id>
+viam machines status --organization=<org id> --location=<location id> --machine=<machine id or name>
```

Comment:
I think for these templates it almost makes more sense to remove location and org, and only show machine id, since you don't need all three at once, *but* we've hitherto said that the templates should show *all* flags even if you wouldn't use them together. What do you think?